lldb/source/Core/FormatEntity.cpp

```diff
@@ -1684,10 +1684,11 @@ bool FormatEntity::Format(const Entry &entry, Stream &s,
       StackFrame *frame = exe_ctx->GetFramePtr();
       if (frame) {
         const Address &pc_addr = frame->GetFrameCodeAddress();
-        if (pc_addr.IsValid() || frame->IsSynthetic()) {
+        if (pc_addr.IsValid()) {
           if (DumpAddressAndContent(s, sc, exe_ctx, pc_addr, false))
             return true;
-        }
+        } else if (frame->IsSynthetic())
+          return true;
```

Comment:
Is returning true here right?  That would mean if I had `{pc: ${frame.pc}}` in my frame format, for a synthetic frame without a PC, this would still print the "pc: " part, which isn't correct.